src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -51,6 +51,18 @@ Follow the rest of the steps on this page within Ubuntu.
 
 </details>
 
+<details>
+
+<summary> Arch Linux </summary>
+
+1. Run `pacman -S --needed git lsb-release cmake swig glu glib2 freeimage freetype2 libxml2 libzip boost-libs gd libgcrypt libzip readline pbzip2 pciutils wget zip unzip jdk-openjdk libusb` to obtain the required dependencies.
```

Comment:
This command is kinda long and might look better in a code block:

```suggestion
1. Run the following to obtain the required dependencies.
     ```bash
     pacman -S --needed git lsb-release cmake swig glu glib2 freeimage freetype2 libxml2 libzip boost-libs gd libgcrypt libzip readline pbzip2 pciutils wget zip unzip jdk-openjdk libusb
     ```
```